docs/guide/browser/assertion-api.md

```diff
@@ -1046,3 +1046,216 @@ await expect.element(queryByTestId('parent')).toHaveSelection('ected text')
 await expect.element(queryByTestId('prev')).not.toHaveSelection()
 await expect.element(queryByTestId('next')).toHaveSelection('ne')
 ```
+
+## toMatchScreenshot <Badge type="warning">experimental</Badge> <Version>3.3.0</Version> {#tomatchscreenshot}
```

Comment:
I guess this version will have to change 😛

Will do before merging, note that this version is written in `config.md` too.